app/services/parsers/bandm/model1.js

```diff
@@ -1,43 +1,52 @@
-const CombineParser = require("../../parser-combine");
-const ParserModel = require("../../parser-model");
+const combine_parser = require("../../parser-combine");
```

Comment:
this change triggers issue for local naming issue on [SonarCloud](https://sonarcloud.io/project/issues?issueStatuses=OPEN%2CCONFIRMED&sinceLeakPeriod=true&branch=feature%2F428101-refactor-models-for-multiple-sheets&id=trade-exportscore-plp)